exporter/datadogexporter/README.md

```diff
@@ -19,6 +19,8 @@
 
 > Please review the Collector's [security documentation](https://github.com/open-telemetry/opentelemetry-collector/blob/main/docs/security-best-practices.md), which contains recommendations on securing sensitive information such as the API key required by this exporter.
 
+> The Datadog Exporter now skips APM stats computation by default. It is recommended to only use the Datadog Connector in order to compute APM stats.
+
```

Comment:
Can you please include example how to turnoff feature gate ? It would be helpful for customers to easily migrate to new version.